packages/dds/shared-object-base/api-report/shared-object-base.api.md

```diff
@@ -16,16 +16,17 @@ import { IEventProvider } from '@fluidframework/core-interfaces';
 import { IEventThisPlaceHolder } from '@fluidframework/core-interfaces';
 import { IExperimentalIncrementalSummaryContext } from '@fluidframework/runtime-definitions';
 import { IFluidDataStoreRuntime } from '@fluidframework/datastore-definitions';
-import { IFluidHandle } from '@fluidframework/core-interfaces';
-import { IFluidHandleContext } from '@fluidframework/core-interfaces';
+import { IFluidHandle } from '@fluidframework/core-interfaces/internal';
+import { IFluidHandle as IFluidHandle_2 } from '@fluidframework/core-interfaces';
```

Comment:
This duplication seems bad, but I'm pretty sure the source of the issue is not related to this PR (api-extractor doesn't understand these are the same). I'm not sure if we should care about this or not.

Reply:
Apparently its a known issue.

Reply:
Tracking issue: https://github.com/microsoft/rushstack/issues/4654